Formula/p/profanity.rb

```diff
@@ -25,6 +25,7 @@ class Profanity < Formula
     depends_on "libtool" => :build
   end
 
+  depends_on "libomemo-c" => :build
```

Comment:
Just noting that this actually just disables OMEMO support. The libraries/pkgconfig names are different so build script can't find library, which is why it ends up build-only dependency (i.e. unused).

Can just leave feature disabled for now until supported upstream as will require various modifications.